### DISCUSSION TITLE
Is there a rule for the language of a city name ? (ex: always use the English name)

### DISCUSSION THREAD

**Discussion by RomainJACQUEMIN** [dr5hn/countries-states-cities-database#546] [Q&A]
Hello,
I am currently looking at some Belgian data, and I have trouble to define a golden format for the city name. As an example:
- Brussels is the English name of the city, and it is stored this way in countries-states-cities-database. (So, there is neither in Dutch (Brussel) nor in French (Bruxelles).
-  Antwerp is the English name of the city, and it is *not* stored this way in countries-states-cities-database, it is rather the Dutch version which is used: Antwerpen.
- Liege  is the English name of the city, and it is *not* stored this way in countries-states-cities-database, it is rather the French version which is used: Liège. (note the accent).

So is there a clear guideline for the language the elements (countries, states, cities) are defined? 
(ie: always in english or always in native langage)

Many thanks

**Comment by dr5hn** [comment]
@Romain-at-Jolt Naming standardization is not in force yet. But my plan is to standardize the names in English only and create another column for the native version.

**Reply by RomainJACQUEMIN** [reply]
May I ask how I could participate? 
(Thanks for the quick answer)
